### DISCUSSION TITLE
Default namelist setting for FMTHIST

### DISCUSSION THREAD

**Discussion by cecilehannay** [NCAR/amwg_dev#273] [TF2: Tuning]
This is follow up on Simone post in https://github.com/NCAR/amwg_dev/issues/263#issuecomment-1555248701

Simone would like to clarify the FMTHIST before we start another run.

She would be great to get input on this:

- as mentioned before, the H2O external forcing has to be removed
- external forcings for aircraft emissions should be added.
- I noticed that ``tau_0_ubc`` was set to .true., but it should be .false.
- these settings are different from our chemistry run, so I would like to check with you, are these correct or should be changed?
```
w_limit_tau_without_eff = .false.
gw_lndscl_sgh = .true.
gw_oro_south_fac = 1.d0
```
This is what we have in the MT chemistry run:
```
gw_limit_tau_without_eff = .true.
gw_lndscl_sgh = .false.
gw_oro_south_fac = 2.d0
```

- Currently the FMTHIST compset does not write restart files, but if you change the PE layout, it does work, for example increasing ntasks to 1800 for atm,ocn,ice,rof,lnd

@cecilehannay @JulioTBacmeister @tilmes

**Comment by cecilehannay** [comment]
Reply from Cecile:
- **H2O external forcing**:  the run mentioned by Simone was done with ``cam6_3_110`` before we decide to remove the H2O external forcing. Starting in ``cam6_3_111``, the H2O external forcing have been removed from the namelist for ``chem=ghg_mam4``.
- external forcings for **aircraft emissions**: Please could you provide the setting you want for 1850 and hist use case in this thread? is it something @fvitt could add in the next tag.
- ``tau_0_ubc``: I am a bit confused here. we had ``tau_0_ubc=.true.`` in every L93/MT I have done. Is it incorrect?
- I will let @JulioTBacmeister (or others) address the gw setting questions.

**Comment by tilmes** [comment]
@cecilehannay 
External forcings: I see that many earlier CAM compsets have not used aircraft emissions for external forcings. Though, I don't know why that would not be added. @andrewgettelman do you know why aircraft emissions of SO2 and BC have not been added to usual CAM runs in the past? To me, it makes sense to add them.
These would have to be added: 
'bc_a4 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/historical_ne30pg3/emissions-cmip6_bc_a4_air_vertical_175001-201412_ne30pg3_c20200521.nc'
'num_a4 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/historical_ne30pg3/emissions-cmip6_num_bc_a4_air_vertical_175001-201412_ne30pg3_c20200521.nc'
'SO2 -> /glade/p/cesmdata/cseg/inputdata/atm/cam/chem/emis/historical_ne30pg3/emissions-cmip6_SO2_air_vertical_175001-201412_ne30pg3_c20200522.nc'

Regarding tau_0_ubc: yes we did set this to .true. in the no chemistry runs, but the default chemistry compset is setting it to .false., but this may be a leftover setting from WACCM.  So, I am unsure if this was tested for the MT level and what was used in the past when we did the QBO tuning. @islasimpson do you know if there was a consensus on what this should be? 

The other settings were also in the chemistry compset, again, it may be wrong, but I just would like to double-check, so we can set them consistently in the chemistry / no chemistry versions.

**Reply by islasimpson** [reply]
I think tau_0_ubc should be set to True in the MT configuration.  This is what we did in all of the previous MT tests.

**Reply by dan800** [reply]
Agreed, tau_0_ubc = TRUE would be the default choice for now for both CAM and CAM-CHEM options.

**Comment by tilmes** [comment]
Concerning these setting in the MT model (response from Rolando):
 
gw_lndscl_sgh = .true.
gw_oro_south_fac = 1.d0

are the appropriate settings for WACCM6 with the ridge scheme for OGW.

In particular, gw_oro_south_fac = 2.d0 is obsolete for the new scheme. This setting was used in WACCM4/CCMI to give stronger OGW forcing in the SH and thus a more realistic ozone hole. This was explained in Garcia et al (JAS, 2017). The idea was that OGW excitation was likely underestimated in the old model because it did not take into account the N-S orientation of major orographic features in the SH, and the zonality of the flow.

This should no longer be necessary with the rige scheme, which takes into account both the variance of the unresolved orography _and_ its orientation.

**Reply by tilmes** [reply]
@JulioTBacmeister do you know what this setting should be?  
w_limit_tau_without_eff = .false.

**Comment by andrewgettelman** [comment]
@tilmes, I think the reason we did not add aviation SO2 and BC is because we have a much more sophisticated treatment of aviation emissions available with a contrail parameterization. 

We added Aviation SO4 (there is not much SO2) and BC 10 years ago as part of our contrail work. The contrail parameterization with H2O is part of the standard CAM code now. It would be easy to add back the aviation SO4 and BC as well. We have emission files for these (1x1 gridded from an aviation inventory, monthly is standard). 

It’s not really a ‘significant’ forcing (IMHO), it’s pretty small and well within the noise of other anthropogenic emissions and interannual variability. It’s large given the small size of aviation as a GHG impact though…

But that contrail work is not the same as CMIP6 emissions. So I have no problem in adding 'what CMIP6 does'. 

One note (as above): I think it is wrong to put aviation emissions in as SO2: I think most of it is SO4 because of the high temperature and humidity environment in the exhaust. But maybe I am wrong.

So I think this is fine, and we would just turn off these emissions when doing contrail studies with the more detailed treatments.

**Comment by dan800** [comment]
All, I diffed a recent FLT namelist (the one with effgw_beres_dp = 0.30D0) with the namelist from the FMT just submitted by @cecilehannay.

">" is the FMT

Click on screenshot to expand (formatting did not work with a text paste).

![Screenshot 2023-06-01 at 11 21 15 AM](https://github.com/NCAR/amwg_dev/assets/37961632/da814a7d-6500-4e4c-9861-14936be8f566)

So apart from some additional output, aircraft emissions and setting the UBC for water to zero flux there are a number of differences.

@PeterHjortLauritzen Are the SE split terms what we need - presumably having to run bigger numbers for nsplit adds to the numerical cost over the scaling with levels, but it's necessary even with H-B now on?

@rgar1a @JulioTBacmeister  Diffs in effgw*/gw* terms as they should be? 

Just want to know what has changed and why when we begin to compare MT & LT (cost and performance).

**Reply by dan800** [reply]
BTW, MT crashed again - I think it's got to do with the extra output - mixing in TEM output on zalat grid with standard lat grid output. Maybe just remove that for now so we can get something to look at by tomorrow? Also by not requesting a lot of daily output and keeping the output the same we can get a better estimate of the true model cost differences.
For future runs I would suggest putting daily TEM into its own history file.

**Reply by dan800** [reply]
@rgarc1a - looks like I mistyped your GitHub ID, so adding it here.

**Reply by dan800** [reply]
Suggested new namelist, splitting out daily TEM into h5:

`
mfilt = 0, 5, 20, 40, 12, 365, 1, 1 
nhtfrq = 0, -24, -24, -3, 0, -24, 0, -8760 
ndens = 2, 2, 2, 2, 2, 2, 2, 1 
interpolate_output = .true., .true., .true., .false., .false., .false., .true. 
interpolate_nlat = 192, 192, 192, 192, 192, 192, 192 
interpolate_nlon = 288, 288, 288, 288, 288, 288, 288 
empty_htapes = .true. 

fincl1 = 'ACTNI', 'ACTNL', 'ACTREI', 'ACTREL', 'AODDUST', 'AODVIS', 'AODVISdn','BURDENBC', 'BURDENDUST', 'BURDENPOM', 'BURDENSEASALT', 'BURDENSO4', 'BURDENSOA', 'CAPE', 'CCN3', 'CDNUMC', 'CH4', 'CLDHGH', 'CLDICE', 'CLDLIQ', 'CLDLOW', 'CLDMED', 'CLDTOT', 'CLOUD', 'CMFMC_DP', 'CT_H2O', 'DCQ', 'DQCORE', 'DTCOND', 'DTCORE', 'DTV', 'EVAPPREC', 'EVAPSNOW', 'FCTI', 'FCTL', 'FICE', 'FLDS', 'FLNS', 'FLNSC', 'FLNT', 'FLNTC', 'FLUT', 'FREQZM', 'FSDS', 'FSDSC', 'FSNS', 'FSNSC', 'FSNT', 'FSNTC', 'FSNTOA', 'ICEFRAC', 'LANDFRAC', 'LHFLX', 'LWCF', 'MPDICE', 'MPDLIQ', 'MPDQ', 'MPDT', 'OCNFRAC', 'OMEGA', 'OMEGA500', 'PBLH', 'PHIS', 'PINT', 'PMID', 'PRECC', 'PRECL', 'PRECSC', 'PRECSL', 'PRECT', 'PS', 'PSL', 'PTEQ', 'PTTEND', 'Q', 'QFLX', 'QRL', 'QRS', 'QTGW', 'RCMTEND_CLUBB', 'RELHUM', 'RVMTEND_CLUBB', 'SHFLX', 'SOLIN', 'SST', 'STEND_CLUBB', 'SWCF', 'T', 'TAUX', 'TAUY', 'TFIX', 'TGCLDIWP', 'TGCLDLWP', 'TMQ', 'TREFHT', 'TS', 'TTGW', 'U', 'U10', 'UBOT', 'UTGWORO', 'UTGW_TOTAL', 'V', 'VBOT', 'VTGWORO', 'VTGW_TOTAL', 'WPRTP_CLUBB', 'WPTHLP_CLUBB', 'Z3', 'ZMDQ', 'ZMDT', 'N2O', 'CO2','CFC11','CFC12', 'CLD_MISR','FISCCP1_COSP','CLD_CAL','CLD_MISR','CLDTOT_CAL','CLDHGH_CAL', 'CLDMED_CAL','CLDLOW_CAL','CLMODIS', 'AODVISdn', 'CCN3', 'CDNUMC','OMEGA500'

fincl2 = 'PS','PMID','U','V','T','Q' 

fincl3 = 'PRECT', 'PRECC', 'FLUT', 'U850', 'U200', 'V850', 'V200', 'OMEGA', 'PSL', 'TS','PS'
 
fincl4 = 'PRECC','PRECL' 

fincl5 = 'Uzm','Vzm','Wzm','THzm', 'VTHzm','WTHzm','UVzm','UWzm'

fincl6 = 'Uzm','Vzm','Wzm','THzm', 'VTHzm','WTHzm','UVzm','UWzm'

phys_grid_ctem_nfreq=-6 
phys_grid_ctem_zm_nbas=120 
phys_grid_ctem_za_nlat=90 

fincl7= 'AQSO4_H2O2','AQSO4_O3', 'bc_a1', 'bc_a4', 'dst_a1', 'dst_a2', 'dst_a3', 'ncl_a1', 'ncl_a1', 'ncl_a2', 'ncl_a3', 'pom_a1', 'pom_a4', 'so4_a1', 'so4_a2', 'so4_a3', 'soa_a1', 'num_a1', 'num_a2', 'num_a3', 'num_a4', 'bc_a1SFWET', 'bc_a4SFWET', 'dst_a1SFWET', 'dst_a2SFWET', 'dst_a3SFWET', 'ncl_a1SFWET', 'ncl_a2SFWET', 'ncl_a3SFWET', 'pom_a1SFWET', 'pom_a4SFWET', 'so4_a1SFWET', 'so4_a2SFWET', 'so4_a3SFWET', 'soa_a1SFWET', 'soa_a2SFWET', 'bc_c1SFWET', 'bc_c4SFWET', 'dst_c1SFWET', 'dst_c2SFWET', 'dst_c3SFWET', 'ncl_c1SFWET', 'ncl_c2SFWET', 'ncl_c3SFWET', 'pom_c1SFWET', 'pom_c4SFWET', 'so4_c1SFWET', 'so4_c2SFWET', 'so4_c3SFWET', 'soa_c1SFWET', 'soa_c2SFWET', 'bc_a1DDF', 'bc_a4DDF', 'dst_a1DDF', 'dst_a2DDF', 'dst_a3DDF', 'ncl_a1DDF', 'ncl_a2DDF', 'ncl_a3DDF', 'pom_a1DDF', 'pom_a4DDF', 'so4_a1DDF', 'so4_a2DDF', 'so4_a3DDF', 'soa_a1DDF', 'soa_a2DDF', 'so4_a1_CLXF', 'so4_a2_CLXF', 'SFbc_a4', 'SFpom_a4', 'SFso4_a1', 'SFso4_a2', 'so4_a1_sfgaex1', 'so4_a2_sfgaex1', 'so4_a3_sfgaex1', 'soa_a1_sfgaex1', 'soa_a2_sfgaex1', 'SFdst_a1','SFdst_a2', 'SFdst_a3', 'SFncl_a1', 'SFncl_a2', 'SFncl_a3', 'num_a2_sfnnuc1', 'SFSO2', 'OCN_FLUX_DMS', 'SAD_SULFC', 'SAD_TROP', 'SAD_AERO'
`

**Reply by rgarc1a** [reply]
@rgarc1a is correct. Thanks.
